source/docs/software/basic-programming/programming-leds.rst

```diff
@@ -0,0 +1,114 @@
+Programming LEDs
```

Comment:
Currently, this article only addresses one type of LEDs, the WS2812 and equivalent protocol. There are are other types of LED strips such as single color LED strips (can be controlled with a relay), RGB LED strips (single color at a time, controlled by multiple relays, like https://www.andymark.com/products/rgb-led-light-strip), and individually addressable LED strips that use other protocols like SPI (WS2801).

The article should be either expanded to cover other types of LEDs, or should address that it covers specifically WS2812 type LEDs.